src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -38,6 +38,24 @@ const SECURE_LINK_VIEWS = [
   },
 ]
 
+const configHasInvalidViewIds = (viewIdsInConfig) => {
+  const validViewIds = SECURE_LINK_VIEWS.map(({ id }) => id)
+  const invalidViewIds = viewIdsInConfig.filter(
+    (viewId) => !validViewIds.includes(viewId)
+  )
```

Comment:
💅🏻  [nitpick]

Probably does not have a huge runtime impact, but the time complexity could be reduced by using a Set rather than an array.

```suggestion
  const validViewIds = new Set(SECURE_LINK_VIEWS.map(({ id }) => id))
  const invalidViewIds = viewIdsInConfig.filter(
    (viewId) => !validViewIds.has(viewId)
  )
```

Reply:
Keep forgetting that Set exists. 😅 Didn't realise `Set.has()` is faster than `Array.includes()` though....TIL

Reply:
Yea Sets and Maps (and plain objects to a degree) have O(1) access, but Arrays are O(n) because a lookup is needed.
Arrays have O(1) access by index, but the index is just the position, not an identifier for the element.

i.e. it's like checking `array[index] === foo` versus `for (const element of array) element === foo`